src/Data/OpenAPI/V3_0_0.hs

```diff
@@ -391,7 +392,7 @@ instance ToJSON OpenAPIObject where
 
 instance FromJSON OpenAPIObject where
   parseJSON = withObject "OpenAPIObject" $ \v -> OpenAPIObject
-    <$> v .: "openapi"
+    <$> v .:? "openapi" .!= "2.0"
```

Comment:
Could you describe the rationale behind this? I understand that we'd want to rule out all pre-3.0 specs, but because the [official spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#fixed-fields) does not have info about what this field should contain, I don't think we need to enforce it. The only thing we need to enforce is the field being required. It's a minor thing, though - if there's a good case for keeping this in there then we can.